modules/stanford_earth_migrate_extend/src/Plugin/migrate/process/StanfordEarthFeedDept.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\stanford_earth_migrate_extend\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Looks up the Department or Program that goes with the events feed URL.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "stanford_earth_feed_dept"
+ * )
+ */
+class StanfordEarthFeedDept extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    $department = NULL;
+    $properties = [
+      'name' => $row->getSourceProperty('current_feed_url'),
+      'vid' => 'stanford_earth_event_feeds',
+    ];
+    $terms = \Drupal::entityTypeManager()
+      ->getStorage('taxonomy_term')
+      ->loadByProperties($properties);
+    if (!empty($terms)) {
```

Comment:
Instead of hardcoding these descriptions here why not always rely on the description field from the UI?

Reply:
Going back to the comment above for the EventsImporterForm, I'm sure there's a better way to match an event's Department/Program to an appropriate taxonomy term, but this works for now.